sequencer/src/main/java/net/consensys/linea/sequencer/txpoolvalidation/LineaTransactionPoolValidatorPlugin.java

```diff
@@ -124,6 +122,12 @@ public void start() {
     }
   }
 
+  @Override
+  public CompletableFuture<Void> reloadConfiguration() {
+    loadDenyListAndRegisterPluginTxValidatorFactory();
```

Comment:
need to look at rejectedTxJsonRpcManager - may need to do something different with that re start/stop lifecycle

Reply:
`factory` field in `TransactionPoolValidatorServiceImpl` is not `volatile` so the change could not be seen immediately by all thread, said that I prefer that the registration is done one, and the reload only updates the deny list, so for example restricting the moving pieces to a  wrap of the deny list in a mutable container that is set at startup and updated on every reload, without having to redo all the initialization of the plugin